websites/M/Moopa/presence.ts

```diff
@@ -0,0 +1,234 @@
+const presence = new Presence({
+		clientId: "1139510267311562842",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets {
+	Logo = "https://i.imgur.com/BzZw6aQ.png",
+}
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: Assets.Logo,
```

Comment:
Why repeat it when you can just add it here?
```suggestion
			largeImageKey: Assets.Logo,
			startTimestamp: browsingTimestamp,
```